packages/block-library/src/query/editor.scss

```diff
@@ -1,3 +1,13 @@
 .editor-styles-wrapper .wp-block.wp-block-query {
 	max-width: 100%;
 }
+
+.block-library-query-toolbar__popover {
+	.components-number-control {
```

Comment:
Are these styles that should be working by default?

Reply:
Probably yes. I've ping @ItsJonQ about this and will have more feedback if they should come from NumberControl.